components/zeus_ui/events/ui_spawn.sqf

```diff
@@ -89,7 +89,7 @@ case "ui_spawn": {
 				_dir = getDir curatorCamera;
 
 				// Tell the server to spawn the group
-
+				f_fnc_server_spawnGroup = compile preprocessFileLineNumbers "components\zeus_ui\fn_server_spawnGroup.sqf";
```

Comment:
Was there an issue with removing line 92? Worked fine in all testing sessions and as Cre8or said, it is redundant as far as I can tell.

Reply:
The last common commit between this framework and the regular one is from years ago, all effort to keep zeus UI up to date has been manual and based on reading commit notes etc, so that's kind of the answer.  The reason why this hasn't been removed is that it hasn't been removed.  Appreciate it being picked up on, I should have time to review and accept these changes soon.